login.css

```diff
@@ -79,53 +53,65 @@ nav a{
 }
 
 
+ul li ul{
+	padding: 0;
+	position: absolute;
+	top: 58px;
+	left: 130px;
+	width: 100px;
+	display: none;
+	visibility: hidden;
+}
 
-.content_1{
+/* Set up position for content div to hold the login function */
+.content{
```

Comment:
01/08/2021: Base on the layout 2, I divided the login page two 2 part. This div is the div box for the login form and it is in the left.